packages/snap/src/index.ts

```diff
@@ -96,13 +96,19 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ origin, request }) =>
             }
           })
         } catch (error) {
+
+          let message = error;
+          if (error.code === 'INSUFFICIENT_FUNDS') {
```

Comment:
```suggestion
          if (error instanceOf Error && error.code === 'INSUFFICIENT_FUNDS') {
```

Reply:
Except this won't work bc Error doesn't have `.code`. What type are you expecting this to be here?